ci/do_ci.sh

```diff
@@ -441,6 +441,21 @@ case $CI_TARGET in
         collect_build_profile coverage
         ;;
 
+    cpu-detection)
+        # this can be removed once the expectation is that the integration test
+        # is present
+        if [[ ! -f "test/server/cgroup_cpu_simple_integration_test.cc" ]]; then
+            echo "CPU detection skipped, no integration test available"
+            exit 0
+        fi
+        setup_clang_toolchain
+        # possibly check the container is doing the right thing at this point
```

Comment:
@premnathnenavath given offline conversation - i think probably lets just remove this comment